tensorflow/contrib/tpu/python/tpu/tpu_constant.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===================================================================
+
+"""TPU constant configs."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.contrib.learn.python.learn.estimators import run_config as run_config_lib
+
+# pylint: disable=protected-access
+_TF_CONFIG_ENV = run_config_lib._TF_CONFIG_ENV
+_SERVICE_KEY = run_config_lib._SERVICE_KEY
+_TPU_WORKER_JOB_NAME = 'tpu_worker_job_name'
+
+
+_INITIAL_LOSS = 1e7
+_ZERO_LOSS = 0.
+_TPU_ESTIMATOR = 'tpu_estimator'
+_ITERATIONS_PER_LOOP_VAR = 'iterations_per_loop'
+_BATCH_SIZE_KEY = 'batch_size'
+_CROSS_REPLICA_SUM_OP = 'CrossReplicaSum'
+_RESERVED_PARAMS_KEYS = [_BATCH_SIZE_KEY]
+
+
+# TODO(b/65703635): Flip the value and remove all dead code.
+_WRAP_INPUT_FN_INTO_WHILE_LOOP = False
```

Comment:
All should be without the underscore

Reply:
I have found in many cases, variable is declared with underscore since keep the same. However, I have changed the variable name as `_WRAP_INPUT_FN_WHILE_LOOP`.